### DISCUSSION TITLE
pyhwp 라이브러리는 우분투에서도 정상적으로 작동 하나요?

### DISCUSSION THREAD

**Discussion by 9mean2** [mete0r/pyhwp#229] [General | 자유게시판]
현재 ubuntu os를 사용 중이고, python 에서는 conda 환경에서 개발 하는 중입니다.

현재 conda env 경로에서 pip로 pyhwp를 설치 하였었는데, 계속 모듈을 찾을 수 없다는 에러만 나오네요 

혹시 해결 방법을 아시는 분이 게실까요

**Comment by mete0r** [comment]
pip 설치 명령과 발생한 에러 메시지를 알려주세요. 그런데 콘다 환경에서도 패키지 설치시 pip를 쓰나요?

**Reply by 9mean2** [reply]
네, pip로 설치 하였습니다.

오류가 나던 이유가, pyhwp만 유독 다른 경로에 설치가 되어서, 따로 환경변수를 추가하여 해결할 수 있었습니다.

**Reply by 9mean2** [reply]
아, 그리고 한가지 더 궁금한 점이 있는데 현재 hwp -> html은 이용 해봤는데 현재 pyhwp로 html -> hwp가 가능할까요?

**Reply by mete0r** [reply]
아니요 html to hwp는 불가능합니다.

**Reply by 9mean2** [reply]
안녕하세요. 궁금한 점이 생겨서 또 한번 찾아오게 됐습니다. 
html to hwp가 불가능 한 이유를 혹시 알 수 있을까요?
